Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM golang
```

Comment:
I think we should be explicit about the version here. We currently require go 1.7, and since there a few patch versions we should go with `FROM golang:1.7.4`.